src/operator/optimizer_op-inl.h

```diff
@@ -1563,6 +1563,192 @@ inline void AdamUpdateEx(const nnvm::NodeAttrs& attrs,
   }
 }
 
+struct LambUpdatePhaseOneParam : public dmlc::Parameter<LambUpdatePhaseOneParam> {
+    float beta1;
+    float beta2;
+    float epsilon;
+    float t;
```

Comment:
@eric-haibin-lin @access2rohit I find this issue when reading the code. Here, the `t` should be the number of updates and should not be stored as `float`, which will lose the precision. I think we need to store it as index_t.

Reply:
we are using float here for integer data type. @sxjscience can you explain how we will loses precision for the operation `beta^t` ?